test/regression/sql/json_operator_support.sql

```diff
@@ -0,0 +1,20 @@
+-- Create Table
+-- Create table for JSON testing
+CREATE TABLE test_json (
```

Comment:
Let's call this `test_jsonb`. I think it would be good to also have the same test for the actual postgres `JSON` type.